src/data/users.js

```diff
@@ -0,0 +1,146 @@
+const mongoCollections = require('../config/mongoCollections');
+const boards = mongoCollections.boards;
+const users = mongoCollections.users;
+const {user_colors} = require('../public/constants/index');
+const {ObjectId} = require('mongodb');
+const bcrypt = require('bcrypt');
+const saltRounds = 16;
+const error_handler = require('../errors/error-handler'); 
+
+
+module.exports = {
+  /**
+   * Get a user with a specific ID.
+   * @param {String} id The user's id.
+   * @returns A user.
+   */
+  readByID: async (id) => {
+    if(!error_handler.checkObjectId(id))
+      throw new Error("id is not valid.");
+
+     const userCollection = await users();
+     const user = await userCollection.findOne({_id: ObjectId(id)});
+     if(user === null)
+       throw new Error("There is no user with that id.");
+ 
+     user['_id'] = user['_id'].toString();
+ 
+     return user;
+  },
+
+  /** Don't need a readAll for users? */
```

Comment:
maybe like a getAllUsersOnBoard ? not sure, just a thought.

Reply:
It'll be too slow